Meta/build-root-filesystem.sh

```diff
@@ -25,7 +25,9 @@ umask 0022
 
 printf "installing base system... "
 cp -R "$SERENITY_ROOT"/Base/* mnt/
-cp -R Root/* mnt/
+#cp -R Root/* mnt/
+(cd Root && tar cf - .) | (cd mnt/ && tar xvf - > /dev/null)
```

Comment:
It copies all files in "Root" (where the build system puts files) into "mnt" (where the image file is mounted). The idea is that tar's directory traversal might be more efficient than "cp -R"'s. It doesn't make much of a difference though. (It's a somewhat common technique for copying directories across ssh pipes, since it makes it easy to add compression.)